lib/Position/WeatherParserUtils.js

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
'use strict'?

Reply:
Also I would move WeatherParserUtils out of Position dir, we will probably also want to parse position-less weather. Maybe /lib/Utils would be better place

Reply:
Can position-less weather exist?  I'm not sure it can.

Reply:
Looking at APRS Protocol Reference it can (i.e. "_10090556c220s004g005t077r000p000P000h50b09900wRSW"). Not sure whether it's used in practice.